java/cuvs-java/pom.xml

```diff
@@ -24,6 +24,64 @@
     <!-- NOTE: The version automatically gets updated when ci/release/update-version.sh is invoked. -->
     <!--CUVS_JAVA#VERSION_UPDATE_MARKER_START--><version>25.02.0</version><!--CUVS_JAVA#VERSION_UPDATE_MARKER_END-->
     <name>cuvs-java</name>
+    <description>
+        This project provides Java bindings for cuVS, enabling approximate nearest neighbors search and clustering
+        on the GPU. As this is still under development, some APIs may change until the 1.0 release.
```

Comment:
Please don't include "still under active development" because it'll always be under active development. Use the term "experimental" instead. 

Also, the release version is 25.02.01. Please do not include an exact release here.